android/tangram/src/com/mapzen/tangram/HttpHandler.java

```diff
@@ -28,8 +30,9 @@ public HttpHandler() {
      * @return true if request was successfully started
      */
     public boolean onRequest(String url, Callback cb) {
-        Request request = okRequestBuilder.tag(url).url(url).build();
-        okClient.newCall(request).enqueue(cb);
+        Request request = okRequestBuilder.url(url).build();
+        Call call = okClient.newCall(request);
```

Comment:
nitpick. Can you collapse lines 34 and 35? (referring to the previous implementation).

I guess you had this when you were explicitly maintaining the list of calls.